.github/dependabot.yml

```
@@ -0,0 +1,8 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "03:00"
```

Would be cooler at funny number 
```suggestion
    time: "04:20"
```